src/components/bulk-email-tool/bulk-email-form/test/BulkEmailForm.test.jsx

```diff
@@ -20,20 +21,25 @@ const dispatchMock = jest.fn();
 
 const tomorrow = new Date();
 tomorrow.setDate(new Date().getDate() + 1);
+const courseMode = courseModeFactory();
 
 function renderBulkEmailForm() {
   const { cohorts } = cohortFactory.build();
   return (
     <BulkEmailProvider>
-      <BulkEmailForm courseId="test" cohorts={cohorts} />
+      <BulkEmailForm
+        courseId="test"
```

Comment:
Just a suggestion: let's come up with a more meaningful name for this identifier 💯

Reply:
Do you have any examples? I used the same name as in the backend.

Reply:
`mockCourseId` or `exampleCourseId`

Reply:
Ok, if it necessary. This ID is being here before I made any change to the tests so I didn't check this part for correctness.

Reply:
I decided to not change `courseId` value to save the common style of test values.